src/app/components/book/book.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-book',
+  standalone: true,
+  imports: [],
+  templateUrl: './book.component.html',
+  styleUrl: './book.component.scss',
+})
+export class BookComponent {
+  mockComments = [
+    'This was a really good book!',
+    "I can't believe that happened!",
```

Comment:
mix of '' and "" ?

Reply:
I assume this is just mock data so its not really a big deal, but maybe configure your prettier or ESLint for this stuff

Reply:
Thank you, don't know why my prettier didn't catch this.

Reply:
I see, it's because of the ' in can't, which means I can't wrap the sentence in single quotations.